src/amd/compiler/aco_ir.h

```diff
@@ -998,46 +998,46 @@ struct Block {
    Block() : index(0) {}
 };
 
-typedef uint16_t Stage;
+using Stage = uint16_t;
```

Comment:
Is there a reason to use `using` instead of `typedef`?

Reply:
Generally the consensus is that `using` is strictly better than `typedef` and one should not use `typedef` anymore.

First it is actually readable.
Second it can be templatized.